README.md

```diff
@@ -24,26 +24,39 @@ git clone https://github.com/Thrillberg/imperial.git
 ```
 cd imperial
 ```
-3. Run the Rails setup command:
+3. Begin the PostgreSQL service:
+```
+sudo service postgresql start
+```
+4. In a separate terminal, start a Redis server:
+```
+redis-server
```

Comment:
Similarly, if redis is installed with homebrew then you can do:
```
brew services start redis
```
And not need a separate terminal open. I'd be happy to have such instructions included if they are helpful! 😄

Reply:
Same as the `PostgreSQL` comment

[ef71461](https://github.com/Thrillberg/imperial/pull/682/commits/ef714611f3e1c8fdc8eec61b117aaee750dbad77)